modules/components/tag-input/tag-input.ts

```diff
@@ -929,7 +917,7 @@ export class TagInputComponent extends TagInputAccessor implements OnInit, After
             .of(this.getItemDisplay(item))
             .map(display => this.setInputValue(display))
             .filter(display => this.inputForm.form.valid && !!display)
-            .map((display: string) => this.createTag(fromAutocomplete ? item : display))
+            .map((display: string) => this.createTag(item))
```

Comment:
I understand there is a bug there, but we cannot introduce a further one to fix this so I suppose we need to handle it elsewhere. I hope to have time to take a look soon